src/entities/orderItem.ts

```diff
@@ -0,0 +1,33 @@
+import { Entity, ManyToOne, Property, Ref, Reference } from "@mikro-orm/core";
+import { Order } from "./order";
+import { Product } from "./product";
+
+@Entity()
+export class OrderItem {
+  @ManyToOne(() => Order, {primary: true, ref: true})
+  order!: Ref<Order>;
+
+  @Property({ type: 'uuid' })
+  id: string;
+
+  @Property()
+  title: string;
+
+  @Property()
+  description: string;
+
+  @Property({ columnType: 'float'})
+  price: number;
+
+  @Property()
+  count: number;
+
+  constructor(dto: { product: Product, orderId: string, count: number}) {
+    this.id = dto.product.id;
```

Comment:
S: If the `OrderItem.id` is actually the productId it would be better to name it as `productId`